packages/admin/src/common/contactImport/useContactImportFromCsv.tsx

```diff
@@ -34,26 +41,99 @@ export const useContactImportFromCsv = ({ scope, targetGroupId, refetchQueries }
+                                            id="cometBrevoModule.contactImport.contactAddAlert"
+                                            defaultMessage="The contact will get a double opt-in email to confirm the subscription. After the contact's confirmation, the contact will be added to the corresponding target groups in this scope depending on the contact's attributes. Before the confirmation the contact will not be shown on the contacts page."
+                                        />
+                                    </Alert>
+                                ) : (
+                                    <Alert severity="error" sx={{ marginBottom: 5 }}>
+                                        <FormattedMessage
+                                            id="cometBrevoModule.contactImport.contactNoOptInAlert"
+                                            defaultMessage="No Double Opt-In email will be sent. Please ensure recipients have given their consent before proceeding."
+                                        />
+                                    </Alert>
+                                )}
+                                <CheckboxField
+                                    name="sendDoubleOptIn"
+                                    label={
+                                        <FormattedMessage
+                                            id="cometBrevoModule.contactImport.sendDoubleOptInMail"
+                                            defaultMessage="Send double opt-in email"
+                                        />
+                                    }
+                                    fullWidth
+                                />
+                            </>
+                        );
+                    }}
+                </FinalForm>
+            </DialogContent>
+            <DialogActions>
+                <Button onClick={handleClose}>
+                    <FormattedMessage {...messages.cancel} />
+                </Button>
+                <Button variant="contained" onClick={() => fileInputRef.current?.click()}>
+                    <FormattedMessage id="cometBrevoModule.contactImport.importContacts" defaultMessage="Import Contacts" />
+                </Button>
+            </DialogActions>
+        </Dialog>
+    );
+
     const component = React.useMemo(
-        () => <ContactImportComponent scope={scope} targetGroupId={targetGroupId} fileInputRef={fileInputRef} refetchQueries={refetchQueries} />,
-        [refetchQueries, scope, targetGroupId],
+        () => (
+            <ContactImportComponent
+                scope={scope}
+                targetGroupId={targetGroupId}
+                fileInputRef={fileInputRef}
+                sendDoubleOptIn={sendDoubleOptIn} // Pass the updated state
+                refetchQueries={refetchQueries}
+            />
+        ),
+        [refetchQueries, scope, targetGroupId, sendDoubleOptIn], // Include sendDoubleOptIn in dependencies
```

Comment:
```suggestion
        [refetchQueries, scope, targetGroupId, sendDoubleOptIn],
```
i don't think this comment is necessary

Reply:
8f460e7e82e440358341be8faf12891db6efd901